### ISSUE TITLE
Buffs For UE, Unfallen, Vaulters. Mainly quest focused

### ISSUE THREAD

**Issue by Arpharnator** [captaincobbs/ESG-Mod#331]
Unfallen :
- Center of Light and Life to 30 per pop type
- Harmony of the heart to +7 approval per pop type
- Heart's ward -> costs 1 antimatter
- Heart's hammer -> deals to 37 dmg per second

UE :
- UE baseline Bonus : 10% -> 15% (now applies after everything), extra 0.1 on ind to inf
- Mount of Zelevas : 100 ind per sd, 40 inf per sd, 2% ind per pop type in empire -> 150 ind per sd, 100 inf per sd, 2% ind and inf per pop type in empire
- Cyborg patriots : Extra 20% troop health on empire

Sheredyn :
- Sheredyn pop collection 5 inf per 100 system manpower -> 10 influence per 100 system manpower
- Sheredyn baseline bonus : 20% ship cost reduction -> 20% ship cost reduction, 20 inf per cp destroyed on empire.
- Sheredyn College : Give +30% xp on heroes

Mezari :
- Daily slogan to apply before law, after +100% food bonus
- Who's Yuri gets an additional +1 science per pop.
- Mezari beseline bonus : 10% -> 15%, 100 science per cp destroyed on empire.
- Journey to the stars : 2 -> 3 approval per science building, -30% overpopulation malus, 3 science per pop

Vaulters : 
- Open Source Defense Systems : +5 Science per ship or defense improvement -> +50 science, +0.8% science per ship or defense science improvement

- System science station : 5 approval per anomaly -> 5 approval per anomaly, 30 science per anomaly, 2 science per pop on anomaly

- NAL module thingy : +2 xp per turn on ships -> +15xp per turn on ships

- Raised Opbot level to 9 at start

- Reduced Vaulter quest requirement from 50% to 35% of galaxy explored
- Reduced requirement from 7 curios with hero to 15 curios
- Reduced quest requirement from 4 buildings to 3

**Comment by captaincobbs** [comment]
I can't tell whether this was rejected or not

**Comment by wakystuf** [comment]
If not rejected, we'll also need to know which Project it's supposed to be tagged to, so that I know where to put it on the wiki

**Comment by Arpharnator** [comment]
Vaulters change are completely correct
UE as you know has been changed slightly by chaos, but this was the "base" on which he did
Unfallen should be good, apart from heart's hammer which was changed

**Comment by Arpharnator** [comment]
It was 1.3.0

**Comment by wakystuf** [comment]
@Arpharnator That's helpful; thanks! Can you help us fill in these blanks? (I don't play any of these factions, and am traveling without my main computer that has ES2 on it). Once we get all of these filled out, I'll post to the wiki.

Unfallen :
* Center of Light and Life from [X] to 30 [resource?] per pop type
* Harmony of the Heart from [X] to +7 approval per pop type
* Heart's Ward cost changed from [X] to 1 antimatter
* Heart's Hammer DPS changed from [X} to [Y] (It sounds like no longer 37?)

UE :
* UE baseline Bonus : From 10% to 15% [influence?] , From 0.1 influence per industry to 0.2 [confirm this is what you meant?]
* Mount of Zelevas: From +100 industry and +40 influence per system level and +2% industry per population type in empire to +150 industry and +100 influence per system level, and +2% industry and influence per population type in empire
* Cyborg patriots : From [X} to +20% troop health on empire
Sheredyn :
* Sheredyn baseline bonus : Added +20 influence per CP destroyed on empire (no change to 20% ship cost reduction)
* Sheredyn population collection bonus: From +5 to +10 influence per 100 system manpower
* Sheredyn College : From [X} to +30% XP on heroes [on system?]
Mezari :
* Daily slogan to +40% food bonus on outposts to apply before industrialist level 1 law, after +100% food bonus from boosting outpost
* Who's Yuri quest reward building gets an additional +1 science per pop.
* Mezari baseline bonus : From 10% to 15% [science?],  [added new +]100 science per cp destroyed on empire.
* Journey to the Stars quest reward building: From 2 to 3 approval per science building, added incremental -30% overpopulation malus, 3 science per pop

Vaulters :
* Open Source Defense Systems : From +5 science per ship or defense improvement to +50 science and +0.8% science per ship or defense improvement
* System science station : From +5 approval per anomaly to +5 approval per anomaly,  +30 science per anomaly and +2 science per pop on anomaly
* NAL module: From +2 xp per turn on ships to +15xp per turn on ships
* Raised Opbot from level [X] to 9 when initially recruited
* Reduced Vaulter quest requirement from 50% to 35% of galaxy explored
* Reduced quest requirement from 7 curiosities explored with hero to 15 curiosities (no hero requirement)
* Reduced quest requirement from 4 buildings to 3 [I suspect that requirement is more nuanced than "build 4 buildings on your empire, so can we clarify?]

**Comment by Arpharnator** [comment]
Center of Light and Life from [10] to 30 [resource?] per pop type
Harmony of the Heart from [5] to +7 approval per pop type
Heart's Ward cost changed from [2] to 1 antimatter
Heart's Hammer DPS changed from [X} to [Y] (It sounds like no longer 37?)


Raised Opbot from level [7] to 9 when initially recruited
Reduced quest requirement from 4 buildings to 3 [It's literally just that, for both quests]

UE baseline Bonus : From 10% to 15% [industry] , From 0.1 influence per industry to 0.2 [influence per industry]
Cyborg patriots : From [0%] to +[30%] troop health on empire
Sheredyn College : From [20%] to +30% XP on heroes [everywhere].
Daily slogan to +[50%] food bonus on outposts to apply before industrialist level 1 law, after +100% food bonus from boosting outpost
Mezari baseline bonus : From 10% to 15% [science on empire], [added new +]100 science per cp destroyed on empire.

**Comment by Arpharnator** [comment]
Also, are you documenting for 1.3.0 or 1.3.1?

**Comment by wakystuf** [comment]
OK, great. I updated my summary above, which I think is now complete except for this one:
* Heart's Hammer DPS changed from [X} to [Y] (It sounds like no longer 37?)

I'm documenting this for all versions, but it sounded like this specific set of changes was for 1.3.0. I just missed it in my first batch of 1.3.0 documentation since it wasn't tagged to 1.3.0.

I also already got Chaos's changes to the UE in 1.3.1 (though feel free to let me know if I missed something there). I posted those a week or 2 ago.
